$MirOS: ports/www/dillo/v2/patches/patch-doc_dillo_1,v 1.1 2009/08/24 16:54:23 tg Exp $
--- doc/dillo.1.orig	Mon Jan 18 18:00:19 2010
+++ doc/dillo.1	Sun Feb 21 18:21:23 2010
@@ -67,29 +67,29 @@ Dillo plugin daemon
 .I dpidc 
 Control program for dpid.
 .TP
-.I ~/.dillo/bm.txt 
+.I ~/.etc/dillo/bm.txt 
 User bookmarks
 .TP
-.I ~/.dillo/cookies.txt 
+.I ~/.etc/dillo/cookies.txt 
 Stored cookies
 .TP
-.I ~/.dillo/cookiesrc 
+.I ~/.etc/dillo/cookiesrc 
 Cookie settings
 .TP
-.I ~/.dillo/dillorc 
+.I ~/.etc/dillo/dillorc 
 Configuration file.
 .TP
-.I ~/.dillo/dpid_comm_keys
+.I ~/.etc/dillo/dpid_comm_keys
 Keys used in dpi daemon communication.
 .TP
-.I ~/.dillo/dpidrc 
+.I ~/.etc/dillo/dpidrc 
 Contains name of directory containing dpis, and associates
 dpi files with protocols.
 .TP
-.I ~/.dillo/keysrc 
+.I ~/.etc/dillo/keysrc 
 Keybindings.
 .TP
-.I ~/.dillo/style.css 
+.I ~/.etc/dillo/style.css 
 User style sheet
 .SH SEE ALSO
 .BR wget (1)
